package.json

```diff
@@ -149,42 +205,37 @@
     "settings": {
       "react": {
         "version": "^16.3.0"
+      },
+      "import/resolver": {
+        "@typescript-eslint/parser": [
+          ".ts",
+          ".tsx"
+        ]
       }
     }
   },
   "prettier": {
     "semi": false,
     "singleQuote": true
   },
-  "lint-staged": {
-    "*.js": [
-      "xo --fix",
-      "prettier --write",
-      "git add"
-    ],
-    "*.{json,md}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
   "ava": {
+    "compileEnhancements": false,
+    "extensions": [
+      "ts",
+      "tsx"
+    ],
     "files": [
-      "**/test/**/*.js"
+      "**/test/**/*.tsx"
```

Comment:
Should we include `.ts`?

Reply:
Not sure if there are going to be tests written in `.ts`, but probably a good idea yea.